app/models/base_image.rb

```diff
@@ -39,6 +39,15 @@ def docker_index_url
     end
   end
 
+  def imagelayers_url
+    if remote?
```

Comment:
Yeah, I had the same thought, though wanted to stay with the existing pattern we use for the hub url. I'll refactor when I add the IL links to the application page (and add that missing test).